x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -351,6 +358,20 @@ const UnstyledProcessEventDot = React.memo(
               height={markerSize * 1.5}
               className="backing"
             />
+            {isOrigin && (
```

Comment:
Added a second backing layer for the origin rather than modifying the existing one to maintain the outline effect on top of the more translucent line. If there is a better way to do this lmk.

Reply:
:grey_question:  Looks good. This one will lay on top of the other one - can't remember if that was part of the design or not.